src/pages/ResultsPage/Map.js

```diff
@@ -12,16 +12,47 @@ const Map = ({ waypoints }) => {
       zoom: 6,
       center
     });
-    waypoints.forEach(waypoint => {
-      let position = { lat: waypoint.lat, lng: waypoint.lng };
-      new google.maps.Marker({ position, map: map });
+    var image = {
+          url: 'https://developers.google.com/maps/documentation/javascript/examples/full/images/beachflag.png',
+          // This marker is 20 pixels wide by 32 pixels high.
+          size: new google.maps.Size(20, 32),
+          // The origin for this image is (0, 0).
+          origin: new google.maps.Point(0, 0),
+          // The anchor for this image is the base of the flagpole at (0, 32).
+          anchor: new google.maps.Point(0, 32)
+    };
+    var shape = {
+          coords: [1, 1, 1, 20, 18, 20, 18, 1],
```

Comment:
What's this do?

Reply:
This sets the image to the url given, and gives it a specific size since it's a png. It also set's the origin which idk what its for, but the anchor sets the point for where exactly the image will be placed. That's the spot of the image that will point to the marker on the map, which is the bottom of the flagpole for this image.

This was more just to show that we **can** add images. But this will def change in the future.